### PR TITLE
skip add styles and styling issues in add bar on both android and IOS

### PR DESCRIPTION
**Pull request by wunnath**

### CODE REVIEW

File: sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/skin/BridgeMessageBuilder.java [hunk 1]
```diff
@@ -146,11 +149,23 @@ public static WritableMap buildDiscoveryResultsReceivedParams(JSONArray results)
     return params;
   }
 
+  public static double stringSize(String fontName,int fontStyle,int textSize,String text) {
```
Comment (lental) [inline sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/skin/BridgeMessageBuilder.java]: can you make this private?

File: sdk/react/adBar.js [hunk 2]
```diff
@@ -42,7 +42,7 @@ var AdBar = React.createClass({
     this.props.onPress(BUTTON_NAMES.SKIP);
   },
 
-  generateResponsiveText: function(showLearnMore) {
+  generateResponsiveText: function(showLearnMore,showSkip) {
```
Comment (lental) [inline sdk/react/adBar.js:45]: space after comma

File: sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/skin/BridgeMessageBuilder.java [hunk 3]
```diff
@@ -186,22 +201,31 @@ public static WritableMap buildAdsParams(Object data) {
         params.putArray("icons", icons);
       }
 
-      //hard coded the values for learnMore for now to prevent from the crash and enabling the learnmore button.
-      double title1=74.672;
+      double title1 = stringSize(fontName,fontStyle,textSize,title);
```
Comment (lental) [inline sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/skin/BridgeMessageBuilder.java]: spaces after all commas, please

**Comment by wunnath** [comment]
spacing added

**Pull request merged**